server/routes/private.js

```diff
@@ -49,6 +50,37 @@ router.post('/api/book-appointment', async (req, res) => {
 
 });
 
+router.post('/api/add_recurring_schedule', async (req, res) => {
+    // Validate appropriate parameters are passed into the book appointment endpoint.
+    // Verification to make sure appointments are only made for future dates and that the worker is available during that time will be handled by the front end.
+    const paramSchema = Joi.object({
+        workerTimeslotId: Joi.number().integer().required(),
```

Comment:
The `workerTimeslotId` would be automatically incremented. Thus, I would remove it from the parameter scheme as it does not need to be passed into the endpoint.

Reply:
Thanks @catai9 workerTimeslotID is removed!